public/app/routes/routes.tsx

```diff
@@ -33,7 +33,6 @@ export function getAppRoutes(): RouteDescriptor[] {
     },
     {
       path: '/d/:uid/:slug?',
-      navId: 'dashboards',
```

Comment:
so is the idea we don't set the `navId` in the routes and instead set it in the individual components? what are the pros/cons here? 🤔

Reply:
Well it was needed on Page component, so instead of having it twice, it's now only specified there

Reply:
@ashharrison90 

I agree, but passing it down is not that easy without wrapping things in a new React Context or passing router props. And needs to go up and down, as the AppChrome is above, the Page below. So by having it on Page at least for now is the simplest, but we can revisit that as we explore how the Page component will work from plugins and what customization they will need.